cloud/user-management.md

```diff
@@ -0,0 +1,53 @@
+# User management
+You can manage users for your Timescale Cloud account by navigating to the
+`Members` section in your Cloud console. From here, you can see your FIXME
+
+## About Timescale Cloud user management
+Timescale Cloud allows you to collaborate with other users on your projects.
+When you have created your project, you can add other users so that they can see
+your project in their Timescale Cloud console. You can add, manage, and delete
+users from the `Members` section of the Cloud console.
+
+If you are the creator of the Project, you are assigned an Administrator role,
+and have the ability to add, manage, and delete other users. If you have been
+added to a project, you are assigned a Member role.
+
+## Add or remove members
+You can add members to your project group from the `Members` section in your
+Cloud console.  The new member does not need  to have a Timescale Cloud account
+before you add them. If they do not yet have an account, they are prompted to
+create one.
+
+<procedure>
+
+### Adding members to a project group
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `Members` section.
+1.  Click `Add New Member`.
+1.  Type the email address of the person that you want to add. A confirmation
+    email is sent to the email address you entered.
+1.  Follow the instructions in the confirmation email to confirm the email
+    address. Alternatively, the new member can create or sign in to a Timescale Cloud account with the email address, and accept the invitation by navigating to the `Invitations` section, and clicking `Accept`.
```

Comment:
Heads up: Soon, it can also be clarified here that invites will be able to be accepted via the "accept invite" button in the emails.

And for new Timescale members, once they click the "accept invite" button in their email, they'll be taken to the signup page where they can register. Once clicking the sign up button, they'll be auto-logged in, auto accept the invite, and jump into the project they were invited to.

This implementation is currently being worked on.